.buildkite/ci.mjs

```diff
@@ -401,6 +417,22 @@ function getBuildEnv(target, options) {
     ABI: isMusl ? "musl" : undefined,
     CMAKE_TLS_VERIFY: "0",
   };
+
+  // ASAN configuration
+  if (asan) {
+    env.ENABLE_ASAN_RELEASE = "ON";
```

Comment:
Instead of defining this here, we should add a new `bun run build:asan` that defines these.